src/ai_hawk/llm/llm_manager.py

```diff
@@ -133,7 +145,7 @@ def invoke(self, prompt: str) -> str:
 
 class LLMLogger:
 
-    def __init__(self, llm: Union[OpenAIModel, OllamaModel, ClaudeModel, GeminiModel]):
+    def __init__(self, llm: Union[OpenAIModel, OllamaModel, ClaudeModel, GeminiModel, GroqAIModel]):
```

Comment:
can you update this to base class AIModel and see if that works

Reply:
Could you clarify? From my side, the current setup seems to be working fine.

Reply:
I have made a commit updating this. can you test your branch once

Reply:
Yeah, its working fine !